uri-scheme.md

```diff
@@ -0,0 +1,35 @@
+# URI Scheme
+
+The `wormhole` URI Scheme is used to encode a wormhole code as a URI. This can then be used to generate QR codes, or be opened by the platform URI handler to open a supporting client.
+
+The general format looks like this, and assumes default values for all query fields:
+`wormhole:{code}`
+
+It can be extended by appending a [query string](https://en.wikipedia.org/wiki/Query_string).
+
+## Query fields
+Applications MUST parse all query fields specified below and fail if they contain unknown fields or unsupported parameter values. Query values are [URL encoded](https://en.wikipedia.org/wiki/Percent-encoding).
+
+* `version`: The version of the URI scheme.  No other value than `0` is accepted.
+Default: 0
+
+* `rendezvous`: The rendezvous server and protocol to use, including its port.
+Default: `ws://relay.magic-wormhole.io:4000/v1`
+
+* `type`: The type of operation requested. Valid values are:
+		* `recv`: The URI generating client is asking to send a file and is requesting the URI parsing client to receive a file.
+		* `send`: The URI generating client is asking to receive a file and is requesting the URI parsing client to offer to send a file.
+Default: `recv`
+
```

Comment:
Please add a parameter for the [AppID](https://github.com/magic-wormhole/magic-wormhole-protocols/blob/114ebb88ec6c6d34711d94beb0257b1d6b4cd0f6/server-protocol.md#application-ids=), making it default to file transfer